db/migrate/20200514214910_rename_bookmarks_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RenameBookmarksController < ActiveRecord::Migration[5.2]
+  def up
+    original_controller = 'foreman_tasks_tasks'
+    original_bookmarks = Bookmark.where({ :controller => original_controller })
+    original_bookmarks_names = original_bookmarks.map { |b| b.name }
+    
+    mistake_controller = 'foreman_tasks/tasks'
+    mistake_bookmarks = Bookmark.where({ :controller => mistake_controller })
+    mistake_bookmarks.each do |b|
+      name = b.name
+      name_is_taken = original_bookmarks_names.include? name
+      name = name_is_taken ? "migrated_#{name}" : name
```

Comment:
Should we rename the _`mistake`_ bookmark or the _`original`_ bookmark?

Reply:
I would vote for `changed_` or `lost_`.

Reply:
Maybe even `recovered_` for both the variable and the prefix

Reply:
I would rename the original one (the one with target name). This way the users will get what they created last

Also I would use a similar approach as in https://github.com/theforeman/foreman/pull/7532 and used a short random string instead of the `migrated_` prefix

Reply:
I like the random part. We can also try and remove one of them (the one we're iterating over sounds more reasonable) in case the `query` is the same. If they are different - we will need to prefix.